<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模式</string>
    </property>
    <addaction name="standardMode"/>
    <addaction name="scienceMode"/>
    <addaction name="programmerMode"/>
    <addaction name="date"/>
    <addaction name="money"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>主题</string>
    </property>
    <addaction name="blackTheme"/>
    <addaction name="whiteTheme"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>语言</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionChinese"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="expression">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="showExpression">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="showNumber">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="buttonsWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="SciencePageWidget" name="sciencePage"/>
       <widget class="ProgrammerPageWidget" name="programmerPage"/>
       <widget class="DatePageWidget" name="datePage"/>
       <widget class="MoneyPageWidget" name="moneyPage"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="historyWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>历史记录</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="scrollAreaLayout"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="standardMode">
   <property name="text">
    <string>标准</string>
   </property>
  </action>
  <action name="scienceMode">
   <property name="text">
    <string>科学</string>
   </property>
  </action>
  <action name="programmerMode">
   <property name="text">
    <string>程序员</string>
   </property>
  </action>
  <action name="date">
   <property name="text">
    <string>日期</string>
   </property>
  </action>
  <action name="moneyMode">
   <property name="text">
    <string>货币</string>
   </property>
  </action>
  <action name="blackTheme">
   <property name="text">
    <string>黑</string>
   </property>
  </action>
  <action name="whiteTheme">
   <property name="text">
    <string>白</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="money">
   <property name="text">
    <string>货币</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SciencePageWidget</class>
   <extends>QWidget</extends>
   <header location="global">SciencePage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MoneyPageWidget</class>
   <extends>QWidget</extends>
   <header location="global">MoneyPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgrammerPageWidget</class>
   <extends>QWidget</extends>
   <header location="global">ProgrammerPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatePageWidget</class>
   <extends>QWidget</extends>
   <header location="global">DatePage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
